app/Knight.py

```diff
@@ -0,0 +1,32 @@
+from app.equipment import Armour, Weapon, Potion

```

Comment:
The import statement should use double quotes instead of single quotes to adhere to the code style guideline: 'Make sure you use the double quotes everywhere.'